lib/spdy-transport/protocol/http2/constants.js

```diff
@@ -46,7 +47,7 @@ exports.flags = {
   PRIORITY: 0x20
 };
 
-exports.settings = {
+exports.settings_server = {
```

Comment:
There is another "settings" export below, that overrides this one. That contains one of parameters with another code, so, I renamed this one. Maybe you suggest another solution?